### DISCUSSION TITLE
umi2搭配@umijs/plugin-qiankun

### DISCUSSION THREAD

**Discussion by timorTm** [umijs/umi#6042] [Q&A]
你好，麻烦请教个问题，我在内网环境的时候，使用的是umi2版本，然后安装@umijs/plugin-qiankun插件，之后在对应的config.js文件的plugins写了@umijs/plugin-qiankun，之后启动的时候提示@tmp/qiankunContext,@tmp/qiankunDefer.js这些依赖没有，但是在外网的时候，不会报错，请问怎么解决这个问题呢？

**Comment by sorrycc** [comment]
qiankun 的问题可以在 https://github.com/umijs/qiankun/issues 提。
